.gitlab/ci_cd.yaml

```diff
@@ -44,15 +44,12 @@ variables:
 
     # Push changes to the test Environment
     - |
-      PUSH_FILE=./deployment/${CI_COMMIT_REF_NAME}-push.sh
-      if [ -f "$PUSH_FILE" ]; then
-        ./deployment/${CI_COMMIT_REF_NAME}-push.sh
+      CI_DEPLOY_FILE=./deploy/${VERSION}/ci-deploy.sh
+      if [ -f "$CI_DEPLOY_FILE" ]; then
+        ./deploy/${VERSION}/ci-deploy.sh
       else
-        if $TB_DEPLOY; then
-          tb deploy --populate --wait
-        else
-          tb push --push-deps --only-changes --fixtures --force ${_CI_POPULATE}
-        fi
+        # FIXME add --semver
```

Comment:
Why we need to add --semver? It is not used in tb deploy

Reply:
it doesn't make sense, removing it